atomics/T1033/T1033.yaml

```diff
@@ -20,11 +20,11 @@ atomic_tests:
     command: |
       cmd.exe /C whoami
       wmic useraccount get /ALL
-      quser /SERVER:"${computername}"
+      quser /SERVER:"#{computername}"
       quser
-      qwinsta.exe" /server:${computername}
+      qwinsta.exe" /server:#{computername}
       qwinsta.exe
-      for /F “tokens=1,2” %i in (‘qwinsta /server:${computername} ^| findstr “Active Disc”‘) do @echo %i | find /v “#” | find /v “console” || echo %j > usernames.txt
+      for /F “tokens=1,2” %i in (‘qwinsta /server:#{computername} ^| findstr “Active Disc”‘) do @echo %i | find /v “#” | find /v “console” || echo %j > usernames.txt
```

Comment:
`computer_name`

Also, are those what I call "evil quotes" that are the funky non-ASCII quotes?